src/components/MerchantsPage/gam/MegaGamListItem/MegaGamListItem.tsx

```diff
@@ -85,29 +85,37 @@ const Container = styled.div`
     isActive ? '#f7f7f7' : '#ffffff'};
   display: flex;
   flex-direction: column;
-  margin-bottom: ${({ isActive }: { isActive: boolean }) =>
-    isActive ? '56px' : '0'};
 `;
 
 const ImageContentContainer = styled.div`
   display: flex;
   flex-direction: ${({ isActive }: { isActive: boolean }) =>
     isActive ? 'column' : 'row'};
+  margin-bottom: ${({ isActive }: { isActive: boolean }) =>
+    isActive ? '60px' : '42px'};
+
+  @media (${tabletScreens}) {
+    flex-direction: column;
+  }
 `;
 
 const CampaignImageContainer = styled.div`
   ${({ isActive }: { isActive: boolean }) => {
+    let margin;
     if (isActive) {
-      return `
-      @media (${tabletScreens}) {
-        height: 106px;
-      }
-      `;
+      margin = '0';
+    } else {
+      margin = '16px 16px 0';
     }
 
     return `
-    align-items: center;
-    display: flex;
+      align-items: center;
+      display: flex;
+
+      @media (${tabletScreens}) {
+        height: 106px;
+        margin: ${margin};
```

Comment:
nit(non-blocking): replace longer if else block with ternaries to match the other conditional styles?

Reply:
there ended up being a lot of differences between the active and inactive states here so i didn't use ternaries here. if i did it'd look a bit like:
```
background-image: url('${heroImageUrl}');
      background-size: ${({ isActive }: { isActive: boolean }) =>
      isActive ? 'cover' : 'auto'};
      border-radius: ${({ isActive }: { isActive: boolean }) =>
      isActive ? '0' : '5px'};
      height: ${({ isActive }: { isActive: boolean }) =>
      isActive ? '240px' : '252px'};
      width: ${({ isActive }: { isActive: boolean }) =>
      isActive ? '100%' : '240px'};

      @media (${tabletScreens}) {
        height: ${({ isActive }: { isActive: boolean }) =>
        isActive ? '106px' : '100px'};;
        width: 100%;
      }
```

which is.. hard to read